measure_cocoapod_size.py

```diff
@@ -79,14 +79,14 @@ def InstallPods(cocoapods, target_dir, spec_repos, target_name, mode, pod_source
     if mode == MODE_SWIFT:
       podfile.write('use_frameworks!\n')
     podfile.write('target \'{}\' do\n'.format(target_name))
-    for pod, version in cocoapods.items():
+    for pod, version in list(cocoapods.items()):
```

Comment:
Was wrapping with in a list constructor required? I believe `items` returns an iterable so the `for` loop works without creating a new list (even in python3).